commercetools-convenience/src/main/java/io/sphere/sdk/sequencegenerators/CustomObjectBigIntegerNumberGeneratorConfig.java

```diff
@@ -0,0 +1,49 @@
+package io.sphere.sdk.sequencegenerators;
+
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.models.Base;
+
+import java.math.BigInteger;
+
+/**
+ * Represents the configuration for the Sequential number generator .
+ *
+ * @see CustomObjectBigIntegerNumberGeneratorConfig
+ */
+
+public final class CustomObjectBigIntegerNumberGeneratorConfig extends Base {
+
+    private final SphereClient sphereClient;
+    private final int maxRetryAttempts;
+    private final String container;
+    private final String key;
+    private final BigInteger initialValue;
+
+    CustomObjectBigIntegerNumberGeneratorConfig(final SphereClient sphereClient, final int maxRetryAttempts, final String container, final String key, final BigInteger initialValue){
```

Comment:
I'd do something:
```
/**
* @param maxRetryAttempts <doc>
* ....
*/
```
But do not write comments if they are duplicated from existing comments.